tests/utils/admincli/vmgroup/vmgrouplifecycle.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This util holds various helper methods related to vmgroup to be consumed by testcases.
+// vmgroup creation, deletion or adding, removing and replacing vm from vmgroup is supported currently.
+
+package vmgroup
```

Comment:
Why do we need a separate package for this util? I think we should keep it in admincli package, so that we can use the functions in this util like this: admincli.CreateVmgroup(), which makes more sense than vmgroup.CreateVmgroup()